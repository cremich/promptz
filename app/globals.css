pre,
code {
  word-wrap: break-word !important;
  white-space: pre-wrap !important;
  white-space: -moz-pre-wrap !important;
  white-space: -pre-wrap !important;
  white-space: -o-pre-wrap !important;
  margin: 0;
  padding: 0;
}

code {
  background-color: #f3f3f7;
}

pre > code {
  display: inline-block;
  padding: 1em;
}
