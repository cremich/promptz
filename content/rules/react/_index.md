---
title: "React"
description: "Rules for React development, component patterns, and best practices."
---
