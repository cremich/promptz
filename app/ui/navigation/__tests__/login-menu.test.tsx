import { describe, expect, test } from "@jest/globals";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import LoginMenu from "@/app/ui/navigation/login-menu";

describe("Login Menu ", () => {
  test("renders login menu unchanged", () => {
    const { container } = render(<LoginMenu />);
    expect(container).toMatchSnapshot();
  });
});
