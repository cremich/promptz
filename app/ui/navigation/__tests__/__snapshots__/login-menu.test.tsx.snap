// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Menu  renders login menu unchanged 1`] = `
<div>
  <div
    class="space-x-4"
  >
    <a
      class="hover:bg-neutral-700 px-3 py-2 rounded-md text-sm font-semibold"
      href="/login"
    >
      Log In
    </a>
    <a
      class="hover:bg-neutral-700 px-3 py-2 rounded-md text-sm font-semibold border"
      href="/signup"
    >
      Create Account
    </a>
  </div>
</div>
`;
